tests/validation/tests/v3_api/resource/terraform/k3s/master/variables.tf

```diff
@@ -1,5 +1,4 @@
 variable "access_key" {}
-variable "access_key_name" {}
```

Comment:
I'm not sure what access_key_name vs. access_key is in this case or where they are separately used, but I'm assuming we might need both since we don't want to have the `jenkins-rke-validation` key hardcoded